git_notes/README.md

```diff
@@ -0,0 +1,56 @@
+# Git Notes research
+
+The aim of this research is to provide
+a feedback around [git notes](https://git-scm.com/docs/git-notes) command.
+
+[Examples](./examples) directory contains [git_notes.sh](./git_notes.sh).
+The command adds number of your specific commits into your `git` repository.
```

Comment:
```suggestion
There's a [git_notes.sh](./git_notes.sh), which adds number of your specific commits into your `git` repository.
```

Reply:
I'd love to provide suggestion for your suggestion :D 

There's a []() script, which...

Reply:
suggestion 'commited' :)